raster/r.what/main.c

```diff
@@ -140,15 +140,7 @@ int main(int argc, char *argv[])
     opt.fs = G_define_standard_option(G_OPT_F_SEP);
     opt.fs->guisection = _("Print");
 
-    opt.format = G_define_option();
-    opt.format->key = "format";
-    opt.format->type = TYPE_STRING;
-    opt.format->required = NO;
```

Comment:
The default is required = YES now, so this is technically changing the behavior, but I think that's fine.

Reply:
Makes sense.